### PR TITLE
Use xenial and spectator 0.1.1

### PR DESCRIPTION
**Pull request by dmuino**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ addons:
     sources:
       - ubuntu-toolchain-r-test
     packages:
-      - g++-5
```
Comment (brharrington) [inline .travis.yml]: Is this the default on xenial?

File: lib/logger.cc [hunk 2]
```diff
@@ -18,7 +18,6 @@ LogManager& log_manager() noexcept {
 LogManager::LogManager() noexcept {
   try {
     logger_ = spdlog::create_async_nb<spdlog::sinks::ansicolor_stdout_sink_mt>(kMainLogger);
-    logger_->set_level(spdlog::level::debug);
```
Comment (brharrington) [inline lib/logger.cc]: Was this causing unwanted noise or what was the reason?

Comment (dmuino) [reply lib/logger.cc]: The default level is info everywhere else, and it was requested we be less noisy.

File: .travis.yml (hunk 1 above)
Comment (dmuino) [reply .travis.yml]: Correct. xenial ships with g++ 5 (and the clang on the travis xenial image is clang 7)

**Pull request merged**